crates/holochain/tests/tests/new_lair/mod.rs

```diff
@@ -85,13 +85,10 @@ async fn test_new_lair_conductor_integration() {
     stdin.write_all(b"passphrase").await.unwrap();
     drop(stdin);
 
-    // cribbed this from test_utils... probably something better would be better
-    if let Ok(status) = tokio::time::timeout(std::time::Duration::from_secs(1), child.wait()).await
-    {
-        panic!("failed to start holochain: {:?}", status);
-    }
-
-    let (mut client, rx) = websocket_client_by_port(ADMIN_PORT).await.unwrap();
```

Comment:
The test was failing here often. The logic above wasn't waiting for Holochain to be ready and so the client couldn't connect. Retrying connecting the admin client is good enough logic for this test.